src/canvas/SelectableCanvas.ts

```diff
@@ -1237,8 +1255,13 @@ export class SelectableCanvas<
   }
 
   protected _createCacheCanvas() {
-    this.cacheCanvasEl = this._createCanvasElement();
-    this.contextCache = this.cacheCanvasEl.getContext('2d')!;
+    this.pixelFindCanvasEl = this._createCanvasElement();
+    this.pixelFindContext = this.pixelFindCanvasEl.getContext('2d', {
+      willReadFrequently: true,
+    })!;
+    const retina = this.getRetinaScaling();
+    this.pixelFindContext.scale(retina, retina);
+    this.setTargetFindTolerance(this.targetFindTolerance);
```

Comment:
setting width/height with this methof after setting the scale will kill the transform? or kill just the pixels? i don't remember

Reply:
excellent question...
I will reorder to be safe
I have no idea